src/util/message.cpp

```diff
@@ -68,6 +68,8 @@ messaget::~messaget()
 {
 }
 
+messaget::eomt messaget::eom;
```

Comment:
Is this explicit instantiation actually required, given the member is already marked `static`?

Reply:
Not for clang nor gcc, but Visual Studio wants it.

Reply:
Would you mind adding a comment? This single line in the middle of lots of other code is surprising, even more so as it would have seemed unnecessary.

Reply:
done